### PR TITLE
fix tire shop having no tools

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
What do you mean tire shop has no tools whatsoever
#### Describe the solution
Add tools that may be in tire shop
#### Describe alternatives you've considered
Revamp the location

### CODE REVIEW

**Comment by akrieger** [comment]
Do tire shops routinely carry angle grinders? That doesn't seem to fit.

**Comment by GuardianDll** [comment]
i suppose tire shops may have a bunch of tools except of tires, including angle grinder
i can remove it of course

**Comment by akrieger** [comment]
Angle grinder maybe for a body shop where they have to take off wrecked panels. Dunno about tire shops that deal with just taking wheels on and off cars that are otherwise totally functional. At least stateside that's all tire shops deal with.

File: data/json/itemgroups/tools.json [hunk 1]
```diff
@@ -430,6 +430,26 @@
...
+    "ammo": 100,
+    "items": [
+      { "item": "jack", "prob": 100 },
+      { "item": "lug_wrench", "prob": 80 },
+      { "item": "jack_small", "prob": 80 },
+      { "item": "grinder_cutoff_disc", "prob": 80 },
+      { "item": "grinder_cutoff_disc", "prob": 80 },
+      { "item": "grinder_cutoff_disc", "prob": 80 },
+      { "item": "angle_grinder", "prob": 20, "charges": [ 0, 500 ] },
+      { "item": "grinder_cutoff_disc", "prob": 20 },
```
Comment (GuardianDll) [inline data/json/itemgroups/tools.json:446]:
```suggestion
```

**Comment by Zireael07** [comment]
@akrieger Same in my corner of the EU

**Pull request merged**